sky/cli.py

```diff
@@ -3539,12 +3566,32 @@ def spot_dashboard(port: Optional[int]):
             try:
                 os.killpg(os.getpgid(ssh_process.pid), signal.SIGTERM)
             except ProcessLookupError:
-                # This happens if spot controller is auto-stopped.
+                # This happens if job controller is auto-stopped.
                 pass
         finally:
             click.echo('Exiting.')
 
 
+# TODO(zhwu): Backward compatibility for the old `sky spot launch` command.
+# It is now renamed to `sky job launch` and the old command is deprecated.
+# Remove in v0.8.0.
+@cli.group(cls=_NaturalOrderGroup)
+def spot():
+    """Alias for Managed Job CLI (default to spot instances)."""
+    pass
+
+
```

Comment:
Also, for this group of aliases, the following message
> WARNING: `sky spot queue` is deprecated and will be removed in a future release. Please use `sky job queue` instead.

isn't entirely accurate. How about something like

> NOTE: `sky spot queue` has been renamed to `sky job queue` and will be removed in a future release. Please use the latter instead.

Reply:
Sounds good. Updated the message, but I would rather keep `WARNING` instead to be more alerting for user to update the command used.